src/pages/PinRelease.tsx

```diff
@@ -29,14 +32,13 @@ export const PinRelease = () => {
                 label="카카오톡"
                 onClick={() => {}}
               />
-              <IconButton variant="square" iconName="link" label="식별자 복사" onClick={() => {}} />
+              <IconButton variant="square" iconName="copy" label="식별자 복사" onClick={() => {}} />
```

Comment:
요것은 코드 복사로 바꾸는게 어떨까용
```suggestion
              <IconButton variant="square" iconName="copy" label="코드 복사" onClick={() => {}} />
```